lego/apps/articles/tests/test_articles_api.py

```diff
@@ -24,6 +24,7 @@ def get_data_with_author(author_pk):
         "description": "good article",
         "content": "the best article",
         "author": author_pk,
+        "youtube_url": "https://www.youtube.com/watch?v=KrzIaRwAMvc",
```

Comment:
As discussed, this can be removed. Both tests add a youtube url manually, and adding it here will affect all other tests using this method.